### PR TITLE
[RW-2100][RISK=LOW]Create Survey Pre-Made Concept Set

### PR DESCRIPTION
**Pull request by NehaBroad**
Screen Shot 2019-08-21 at 12 26 52 PM

### CODE REVIEW

File: ui/src/app/pages/data/data-set/dataset-page.tsx [hunk 1]
```diff
@@ -619,6 +667,26 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                       {plusLink('concept-sets-link', conceptSetsPath, !this.canWrite)}
                     </BoxHeader>
                   <div style={{height: '9rem', overflowY: 'auto'}}>
+                    <Subheader>Prepackaged Concept Set</Subheader>
```
Comment (jaycarlton) [inline ui/src/app/pages/data/data-set/dataset-page.tsx]: I believe Set should be plural for both these titles.

File: api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java [hunk 2]
```diff
@@ -521,4 +539,15 @@ private static String convertNamedParameterToString(
         throw new BadRequestException("Language not supported");
     }
   }
+
+  private ConceptSet handlePrePackagedSurveyConceptSet() {
+    List<Long> conceptIds = conceptBigQueryService.getSurveyQuestionConceptIds();
+    ConceptSet surveyConceptSets = new ConceptSet();
+    surveyConceptSets.setName("PrePackaged Survey");
```
Comment (als364) [inline api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java]: This text should be something more descriptive, like, "All Surveys" or "All Survey Questions".

File: ui/src/app/pages/data/data-set/dataset-page.tsx [hunk 3]
```diff
@@ -347,6 +359,27 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
...
+      if (!selected) {
+        const updatedValueSets =
+            valueSets.filter(valueSet => !(fp.contains(valueSet.domain, domain)));
+        const updatedSelectedValues =
+            selectedValues.filter(selectedValue =>
+                !fp.contains(selectedValue.domain, domain));
+        this.setState({valueSets: updatedValueSets, selectedValues: updatedSelectedValues});
+        return;
+      }
+      const newDomains = [];
```
Comment (als364) [inline ui/src/app/pages/data/data-set/dataset-page.tsx]: Could this be collapsed to `const newDomains = [domain]`?

File: api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java (hunk 2 above)
Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java]: Yes i forgot to change here. it should be All Surveys

**Pull request merged**